.changeset/happy-cups-fry.md

```diff
@@ -0,0 +1,18 @@
+---
+'backend': patch
```

Comment:
```suggestion
'backend': minor
```

This is adding a new nonbreaking feature.

Reply:
Fixed